__tests__/app.test.js

```diff
@@ -212,3 +205,55 @@ describe("GET /api/articles/:articleId", () => {
     });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Return all comments for an article", async () => {
+    const response = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
+    expect(response.body).toHaveLength(11);
+  });
+
+  test("200: Return expected comment data", async () => {
+    const response = await request(app)
+      .get("/api/articles/1/comments")
+      .expect(200);
+    expect(response.body).toBeSortedBy("created_at", { descending: true });
+    response.body.forEach((comment) => {
+      expect(comment).toMatchObject({
+        comment_id: expect.any(Number),
+        votes: expect.any(Number),
+        created_at: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String),
+        article_id: expect.any(Number),
```

Comment:
You know that the article_id you're requesting is 1, so you can be more specific than .any(Number)